setup.cfg

```diff
@@ -1,5 +1,5 @@
 [flake8]
-ignore = 
+ignore =
```

Comment:
what did this?

Reply:
My editor strips trailing white space. Normally flake8 would catch this and yell at whoever added it, but I guess since it's not a python file, it wasn't being checked.